File: LpSolve/Result/AmbigousSeidelResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using LpSolve.Elements;

namespace LpSolve.Result
{
	public class AmbigousSeidelResult : SeidelResult
	{
		public Point[] AmbigousPoints { get; private set; }

		public AmbigousSeidelResult(Point[] points)
		{
			this.AmbigousPoints = points;
		}

		public override SeidelResult Resolve(HalfSpace halfSpace)
		{
			throw new NotImplementedException();
		}
	}
}

File: cpp/Result/AmbigousSeidelResult.cpp
#include "AmbigousSeidelResult.h"

using namespace LpSolveCpp::Elements;
namespace LpSolveCpp
{
	namespace Result
	{

		std::vector<Point*> AmbigousSeidelResult::getAmbigousPoints() const
		{
			return privateAmbigousPoints;
		}

		void AmbigousSeidelResult::setAmbigousPoints(std::vector<Point*> &value)
		{
			privateAmbigousPoints = value;
		}

		AmbigousSeidelResult::AmbigousSeidelResult(std::vector<Point*> &points)
		{
			this->setAmbigousPoints(points);
		}

		SeidelResult *AmbigousSeidelResult::Resolve(HalfSpace *halfSpace)
		{
			throw NotImplementedException();
		}
	}
}
